website/web/__init__.py

```diff
@@ -817,7 +817,7 @@ def web_misp_push_view(tree_uuid: str) -> str | WerkzeugResponse | Response | No
 
 
 @app.route('/tree/<string:tree_uuid>/modules', methods=['GET'])
-def modules(tree_uuid: str) -> str | WerkzeugResponse | Response:
+def modules(tree_uuid: str, mail: bool = False) -> str | WerkzeugResponse | Response:
```

Comment:
This method should only be used to render things on the UI, not in any other case. This is the expectation for all the methods with `@app.route`.

The new human readable response should be a dedicated method in lookyloo, or use the machine readable one (`lookyloo.get_modules_responses(tree_uuid)`), and make it human readable in `mail_send`.

Reply:
changed it now, hope that it works for every module (couldn't test every module).